### PR TITLE
edit: structuring a basic HTML landing page 

### PR DESCRIPTION
**Pull request by giannamoth**
Based on the UX direction I've written down a basic HTM/CSS landing page.
I've shifted away from React, because of the timing and amount of pages to develop I felt more comfortable by doing so.

There are some dummy links, that we can soon test.

my next goals are:
- make it more similar to the one on the UX page visually
- creating and connecting the other pages
- cleaning the code and making it more standardized.

Bitte WB and UX peers, let me know if I can merge it to the `develop` and keep going with the updates 😎

### CODE REVIEW

**Comment by vagechirkov** [comment]
This is how it looks in Google Chrome (I have added the different input form to have the file input):
image

**Comment by vagechirkov** [comment]
The problem that I see so far is that the layout breaks when the resolution is different and for the mobile devices:

See photos:

<details>


image
image

image

</details>

File: node_app/client/index.html [hunk 1]
```diff
@@ -8,76 +8,84 @@
...
-                    <button>Birds</button>
-                    <button>Street</button>
-                    <button>Thunderstorm</button>
-                    <button>Street</button>
+
+                    <span class="badge rounded-pill bg-light">Sea</span>
+                    <span class="badge rounded-pill bg-light">Birds</span>
+                    <span class="badge rounded-pill bg-light">Street</span>
+                    <span class="badge rounded-pill bg-light">Storm</span>
+                    <span class="badge rounded-pill bg-light">Street</span>
```
Comment (vagechirkov) [inline node_app/client/index.html]: I would suggest using bootstrap Button group here https://getbootstrap.com/docs/5.1/components/button-group/

File: node_app/client/index.html [hunk 2]
```diff
@@ -98,4 +189,4 @@ <h5>Upload your track</h5>
```
Comment (vagechirkov) [inline node_app/client/index.html:195]: add Bootstrap JS here 

```suggestion
    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js"
            integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p"
            crossorigin="anonymous"></script>
```

Comment (giannamoth) [reply node_app/client/index.html:195]: hei, I still have to make it Responsive, so far it is just a  workable structure.

I'm gonna add responsiveness at last, when we agree on the style

File: node_app/client/index.html (hunk 1 above)
Comment (giannamoth) [reply node_app/client/index.html]: yes, agreee with this

**Comment by giannamoth** [comment]
in general weird, this is how I see it:

<details>

Cattura3
Cattura1
Cattura2

<\details>

**Comment by giannamoth** [comment]
@vagechirkov  See above. Do you have an idea why you see a different layout than mine?

**Comment by giannamoth** [comment]
I think you are seeing the version from yesterday 🤔

**Comment by vagechirkov** [comment]
Yeah, I think you are right, my bad. I have fetched your fork and it works! Looks amazing! 

<details>

image

<\details>

**Comment by giannamoth** [comment]
supi :) I'm going to adjust the button and stuff before this evening meeting :)
Thx man!!!!

**Comment by giannamoth** [comment]
ok Badges are buttons now, Bootstap JS script is also there. Responsiveness comes int he next days 💪

**Pull request merged**